src/Kafka/CreateKafkaTopic/components/CreateTopicHead.tsx

```diff
@@ -0,0 +1,69 @@
+import type { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  PageSection,
+  PageSectionVariants,
+  Title,
+  Switch,
+  Breadcrumb,
+  BreadcrumbItem,
+} from "@patternfly/react-core";
+import { Link } from "react-router-dom";
+
+export type CreateTopicProps = {
+  showAllOptionsClicked: boolean;
+  kafkaName: string;
+  kafkaPageLink: string;
+  kafkaInstanceLink: string;
+  setshowAllOptionsClicked: (value: boolean) => void;
```

Comment:
It's an event, so it should start with `on`. `onShowAllOptions` would work

Reply:
done